### ISSUE TITLE
Print preview is not opening without any error. 

### ISSUE THREAD

**Issue by arpit9612** [id1945/ngx-print-element#14]
@id1945 Hi,
I am using angular V13, I am trying to print a invoice with all the data being dynamic.
But after multiple attempts while testing the print preview does not open without any error.
The browser just prints the window object that the library prints on closing the preview. I have checked everything but not able to fix this issue. I don't even know that is it happening because of the library or because of some issue in browser but the print preview just stops coming lets say after 50 attempts.

**Comment by id1945** [comment]
@arpit9612 Please record a video so I can see this issue more clearly

**Comment by arpit9612** [comment]
Hi @id1945,
Hope you are doing good. My client raised this issue and I have also checked this, I think the problem is specific to the the chrome browser version 118.0.5993.117. As I have the latest version of chrome and I am not having this issue a single time.
As per your request for the video I am sorry but the project is confidential and the client is not sharing the issue video he is having.
Here is my package.json

{
  "name": "ictservices-portal",
  "version": "1.42.0",
  "scripts": {
    "start": "node server.js",
    "serve": "ng serve",
    "stats": "webpack-bundle-analyzer dist/ictservices-portal/stats.json",
    "build:prod": "ng build --prod",
    "sw:server": "http-server -p 8080 -c-1 dist/ictservices-portal",
    "lint": "tslint --fix \"src/**/*.ts\""
  },
  "private": true,
  "dependencies": {
    "@amcharts/amcharts4": "^4.10.16",
    "@angular-material-components/datetime-picker": "^7.0.1",
    "@angular-material-components/moment-adapter": "^7.0.0",
    "@angular/animations": "13.3.12",
    "@angular/cdk": "13.3.9",
    "@angular/common": "13.3.12",
    "@angular/compiler": "13.3.12",
    "@angular/core": "13.3.12",
    "@angular/forms": "13.3.12",
    "@angular/http": "^7.2.16",
    "@angular/material": "13.3.9",
    "@angular/material-moment-adapter": "13.3.9",
    "@angular/platform-browser": "13.3.12",
    "@angular/platform-browser-dynamic": "13.3.12",
    "@angular/pwa": "^0.1102.2",
    "@angular/router": "13.3.12",
    "@angular/service-worker": "13.3.12",
    "@auth0/auth0-angular": "^2.0.1",
    "@highcharts/map-collection": "^1.1.3",
    "@ng-idle/core": "^10.0.0-beta.1",
    "@ng-idle/keepalive": "^10.0.0-beta.1",
    "@ngx-translate/core": "^13.0.0",
    "@ngx-translate/http-loader": "^6.0.0",
    "@types/node": "^14.14.31",
    "@zxing/ngx-scanner": "^3.1.3",
    "bcryptjs": "^2.4.3",
    "bootstrap": "^4.6.0",
    "compression": "^1.7.4",
    "core-js": "^3.9.1",
    "cryptojs": "^2.5.3",
    "dayjs": "^1.11.9",
    "express": "^4.17.1",
    "hammerjs": "^2.0.8",
    "jwt-decode": "^2.2.0",
    "lib-hierarchy-management": "file:../hierarchy-management-lib/dist/lib-hierarchy-management/lib-hierarchy-management-0.0.1.tgz",
    "lib-portal-login": "file:../portal-login-lib/dist/lib-portal-login/lib-portal-login-0.0.1.tgz",
    "lib-tenant-staff-management": "file:../tenant-staff-management-lib/dist/lib-tenant-staff-management/lib-tenant-staff-management-0.0.1.tgz",
    "moment": "2.29.1",
    "ngx-daterangepicker-material": "^5.0.2",
    "ngx-device-detector": "^2.0.6",
    "ngx-mat-select-search": "5.0.0",
    "ngx-owl-carousel-o": "^6.0.2",
    "ngx-print-element": "2.0.19",
    "path": "^0.12.7",
    "reflect-metadata": "~0.1.13",
    "rxjs": "~6.6.6",
    "webpack": "^5.73.0",
    "xlsx": "^0.16.9",
    "zone.js": "~0.11.4"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "^13.3.10",
    "@angular/cli": "^13.3.10",
    "@angular/compiler-cli": "~13.3.12",
    "@types/crypto-js": "^4.1.1",
    "@types/jasmine": "3.6.4",
    "@types/jasminewd2": "~2.0.8",
    "@types/jwt-decode": "^2.2.1",
    "codelyzer": "~6.0.1",
    "http-server": "0.12.3",
    "jasmine-core": "~3.6.0",
    "jasmine-spec-reporter": "~6.0.0",
    "karma": "~6.3.13",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage-istanbul-reporter": "~3.0.3",
    "karma-jasmine": "~4.0.1",
    "karma-jasmine-html-reporter": "^1.5.4",
    "prettier": "^2.2.1",
    "pretty-quick": "^3.1.0",
    "protractor": "~7.0.0",
    "ts-node": "~9.1.1",
    "tslint": "~6.1.3",
    "tslint-clean-code": "0.2.10",
    "tslint-config-prettier": "^1.18.0",
    "tslint-html-report": "^2.2.0",
    "tslint-sonarts": "^1.8.0",
    "typescript": "~4.6.4",
    "webpack-bundle-analyzer": "^4.4.0"
  }
}

Can you please check this issue regarding the given browser version.
Meantime I'll communicate with client and ask him to share the video if possible.

**Comment by id1945** [comment]
@arpit9612  I updated to the latest version v2.1.3 for angular 6->15 changed a few things maybe this error will go away but that's just my guess. I think you need to reproduce this error on your localhost. And upgrade to v2.1.3 to test the above bug again

**Comment by arpit9612** [comment]
@id1945 Yes I have upgraded to version 2.1.3 but there is some issue I guess as on passing the config and template option to the  print method the print preview shows blank page

**Comment by id1945** [comment]
@arpit9612 Pls check new version
